tests/fiuinfo_test.cc

```diff
@@ -0,0 +1,48 @@
+#include <gtest/gtest.h>
+#include <unistd.h>
+
+#include "utilities/fault_injection.h"
+#include "utilities/utils.h"
+
+std::string fiu_script;
+
+TEST(Fiuinfo, PassInfoWithRun) {
+  EXPECT_TRUE(fiu_fail("fail"));
+  EXPECT_EQ(fault_injection_last_info(), "test");
```

Comment:
Where does "test" come from here?

Reply:
Here: https://github.com/advancedtelematic/aktualizr/blob/a51704b6fac25ce10f1fe024a81a01411225226f/tests/CMakeLists.txt#L77

Reply:
Thanks. It could perhaps be a more greppable string, but not a big deal at all.

Reply:
good point, changed it to something more unique